portal-backend/tests/depmap/data_explorer_2/context/test_custom_operators.py

```diff
@@ -75,6 +77,6 @@ def expressions_are_equivalent(boolean_value, json_logic_expr):
 
     # These expressions don't use variables (just pure logic)
     var_name = "dummy variable"
-    result = ContextEvaluator(context).is_match(var_name)
+    result = ContextEvaluator(context, slice_to_dict).is_match(var_name)
```

Comment:
It really doesn't matter but `lambda: None` would work in place of `slice_to_dict` I think.

This test is a little funny. It wants to test the internal logic engine but that's not exposed so I had to pass a meaningless variable that never gets looked up.